src/index.js

```diff
@@ -15,6 +15,17 @@ module.exports = async function({
   if (!html) {
     throw Error('You must provide an html property.')
   }
+  let screeshotArgs = {};
+  if (/jpe?g/.test(type)) {
```

Comment:
As you can see in the documentation you linked earlier only png and jpeg are supported values. jpg is not valid.

```suggestion
  if (type === 'jpeg') {
```